src/ol/renderer/canvas/canvasvectorlayerrenderer.js

```diff
@@ -169,7 +169,8 @@ ol.renderer.canvas.VectorLayer.prototype.prepareFrame =
       frameState.attributions, vectorSource.getAttributions());
   this.updateLogos(frameState, vectorSource);
 
-  if (!this.dirty_ && (frameState.viewHints[ol.ViewHint.ANIMATING] ||
+  if (!this.dirty_ && (!vectorLayer.getUpdateWhileAnimating() &&
+      frameState.viewHints[ol.ViewHint.ANIMATING] ||
       frameState.viewHints[ol.ViewHint.INTERACTING])) {
```

Comment:
@elemoine If we revisit that, we should do it together with looking at the `loadTilesWhileAnimating` and `loadTilesWhileInteracting` map options.

I have addressed all review comments. Is this good to merge?